fasjson_client/client.py

```diff
@@ -0,0 +1,109 @@
+import errno
+from urllib.parse import urlparse
+
+import yaml
+import requests
+from requests_gssapi import HTTPSPNEGOAuth
+from bravado import requests_client
+from bravado.client import SwaggerClient
+import gssapi
```

Comment:
The convention for the order is `import X` lines first and then `from X import Y` lines.

Reply:
That comment is still valid I think.